README.md

```diff
@@ -1 +1,41 @@
-# lboard
\ No newline at end of file
+# lboard
+
+In this project, I have built a web app powered by the Leaderboard APIs. I've achieved by using the Webpack Framework.
+
```

Comment:
- [ ] **OPTIONAL**
_I'm sorry I didn't mention this point earlier!_
Kindly always keep in mind to attach a screenshot of your app to the README file. 😄